### PR TITLE
AvalancheGo@v1.10.0-fuji

### PR DESCRIPTION
**Pull request by usmaneth**
Included cortina-prerelease-notes (V1.10.0)

### CODE REVIEW

**Review by yulin-dong — CHANGES_REQUESTED** [review]
Please fix lint errors.

File: docs/apis/avalanchego/avalanchego-release-notes.md [hunk 1]
```diff
@@ -6,6 +6,26 @@
 
 :::
 
+## V1.10.0 [View on GitHub](https://github.com/ava-labs/avalanchego/releases/tag/v1.10.0-fuji)
```
Comment (yulin-dong) [inline docs/apis/avalanchego/avalanchego-release-notes.md:9]:
```suggestion
## PRE_RELEASE V1.10.0-fuji [View on GitHub](https://github.com/ava-labs/avalanchego/releases/tag/v1.10.0-fuji)
```

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: docs/apis/avalanchego/avalanchego-release-notes.md [hunk 2]
```diff
@@ -6,6 +6,26 @@
...
+- Linearizes the X-chain 
+- Introduces delegation batching to the P-chain 
+- Increases the maximum block size on the C-chain.
+
+The changes in the upgrade go into effect at 11 AM EST, April 6th 2023 on the Fuji testnet. After Fuji is updated and verified, a mainnet compatible release will be published.
+
+All Fuji nodes should upgrade before 11 AM EST, April 6th 2023.
+
+The supported plugin version is `25.`
+
```
Comment (patrick-ogrady) [inline docs/apis/avalanchego/avalanchego-release-notes.md:26]: Include this:


If you're using the installer script on your **Fuji** node, you'll need to run the following command to upgrade (marked as a pre-release for now):

```
./avalanchego-installer.sh --version v1.10.0-fuji
```

File: docs/apis/avalanchego/avalanchego-release-notes.md [hunk 3]
```diff
@@ -6,6 +6,26 @@
...
+:::
```
Comment (patrick-ogrady) [inline docs/apis/avalanchego/avalanchego-release-notes.md:27]: what is this for?

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: docs/apis/avalanchego/avalanchego-release-notes.md [hunk 4]
```diff
@@ -6,6 +6,26 @@
...
```
Comment (yulin-dong) [inline docs/apis/avalanchego/avalanchego-release-notes.md:27]:
```suggestion
```

**Review by yulin-dong — APPROVED** [review]

File: docs/apis/avalanchego/avalanchego-release-notes.md [hunk 5]
```diff
@@ -6,6 +6,30 @@
...
+
+Please note that this release is unable to run Mainnet - and will display "Mainnet is not supported" if attempted to run with a Mainnet configuration. 
+
+This upgrade: 
+
+- Linearizes the X-chain 
+- Introduces delegation batching to the P-chain 
+- Increases the maximum block size on the C-chain. 
+
+The changes in the upgrade go into effect at 11 AM ET, April 6th 2023 on the Fuji testnet.
```
Comment (yulin-dong) [inline docs/apis/avalanchego/avalanchego-release-notes.md:21]:
```suggestion
The changes in the upgrade go into effect at 11 AM EDT, April 6th 2023 on the Fuji testnet. 
```

File: docs/apis/avalanchego/avalanchego-release-notes.md [hunk 6]
```diff
@@ -6,6 +6,30 @@
...
+
+- Linearizes the X-chain 
+- Introduces delegation batching to the P-chain 
+- Increases the maximum block size on the C-chain. 
+
+The changes in the upgrade go into effect at 11 AM ET, April 6th 2023 on the Fuji testnet. 
+
+After Fuji is updated and verified, a Mainnet compatible release will be published. 
+
+All Fuji nodes should upgrade before 11 AM ET, April 6th 2023.
```
Comment (yulin-dong) [inline docs/apis/avalanchego/avalanchego-release-notes.md:25]:
```suggestion
All Fuji nodes should upgrade before 11 AM EDT, April 6th 2023. 
```

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: docs/apis/avalanchego/avalanchego-release-notes.md [hunk 7]
```diff
@@ -6,6 +6,33 @@
...
+## PRE_RELEASE V1.10.0-Fuji [View on GitHub](https://github.com/ava-labs/avalanchego/releases/tag/v1.10.0-fuji)
+
+**Cortina Pre-release (v1.10.0-Fuji)** 
+
+Please note that this release is unable to run Mainnet - and will display:
+"Mainnet is not supported" if attempted to run with a Mainnet configuration. 
+
+This upgrade: 
+
+- Linearizes the X-chain
```
Comment (yulin-dong) [inline docs/apis/avalanchego/avalanchego-release-notes.md:18]: Please add Linearize to `.github/styles/Vocab/Products/accept.txt`

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: docs/apis/avalanchego/avalanchego-release-notes.md [hunk 8]
```diff
@@ -6,6 +6,33 @@
...
+The changes in the upgrade go into effect at 11 AM ET, April 6th 2023 on the Fuji testnet. 
+
+After Fuji is updated and verified, a Mainnet compatible release will be published. 
+
+All Fuji nodes should upgrade before 11 AM ET, April 6th 2023. 
+
+If you're using the installer script on your Fuji node, you'll need to run 
+the following command to upgrade (marked as a pre-release for now):
+
+`./avalanchego-installer.sh --version v1.10.0-Fuji`
```
Comment (patrick-ogrady) [inline docs/apis/avalanchego/avalanchego-release-notes.md:31]: Surround this with ``` instead

**Review by yulin-dong — APPROVED** [review]

**Pull request merged**